.github/workflows/verify-build.yaml

```diff
@@ -0,0 +1,26 @@
+name: Verify Build
```

Comment:
not opposed to it, but is this action necessary? if you validate on build, you won't need to invoke this on the fly with PRs.

Reply:
yes, we can verify it after web build script as well. already moved there.